resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/Utils.scala

```diff
@@ -131,4 +133,9 @@ object Utils extends Logging {
         s"under spark home test dir ${sparkHomeDir.toAbsolutePath}!")
     }
   }
+
+  def isHadoop3(): Boolean = {
```

Comment:
Out of curiosity, can we `org.apache.hadoop.util.VersionInfo`? e.g.) `VersionInfo.getVersion.startsWith("3")`. At least I know this is being used in Apache Spark codebase.

Reply:
Sure, Thanks.